command/acl/token/formatter.go

```diff
@@ -173,13 +173,9 @@ func (f *prettyFormatter) formatTokenListEntry(token *api.ACLTokenListEntry) str
 		}
 	}
 	if len(token.NodeIdentities) > 0 {
-		buffer.WriteString(fmt.Sprintln("Service Identities:"))
-		for _, svcid := range token.ServiceIdentities {
-			if len(svcid.Datacenters) > 0 {
-				buffer.WriteString(fmt.Sprintf("   %s (Datacenters: %s)\n", svcid.ServiceName, strings.Join(svcid.Datacenters, ", ")))
-			} else {
-				buffer.WriteString(fmt.Sprintf("   %s (Datacenters: all)\n", svcid.ServiceName))
-			}
+		buffer.WriteString(fmt.Sprintln("Node Identities:"))
+		for _, nodeid := range token.NodeIdentities {
+			buffer.WriteString(fmt.Sprintf("   %s (Datacenter: %s)\n", nodeid.NodeName, nodeid.Datacenter))
```

Comment:
i may be wrong here but I think this is breaking some of the units 🤔 

re-running the current failed ones.

Reply:
The failing tests seem to be related to anti-entropy sync, whereas this PR only changes the output of one of the CLI commands. I'm also able to reproduce the test failures on the `main` branch, so I suspect that they're unrelated to these changes.